src/QsCompiler/CompilationManager/FileContentManager.cs

```diff
@@ -32,6 +32,11 @@ public class FileContentManager : IDisposable
 
         public string FileName { get; }
 
+        /// <summary>
+        /// Indicates the current version of the document according to the Language Server Protocol.
```

Comment:
Since FileContentManager isn't LSP-specific, maybe it would be better not to refer to LSP here - something like this?

```suggestion
        /// An arbitrary integer representing the current version number of the file, or null if no version number is available.
        /// The version number may change at any time to any other integer, including a lower number than its current value.
```

Reply:
Sounds good. Applying.